decentraland/constants/UserActionStatus.js

```diff
@@ -0,0 +1,17 @@
+// To be used for meta-tx
```

Comment:
Also for contract-based accounts with signed messages that will be bundled into tx with enough signatures (which is a meta-tx in a sense but I'd argue not what people normally mean by meta-tx

Reply:
```suggestion
// To be used for meta-tx
// Also for contract-based accounts with signed messages
// that will be bundled into tx with enough signatures 
```

Reply:
Want to add this in in your current PR?

Reply:
Done.

Reply:
Thanks! Resolving this